tests/elemwise_func_test.py

```diff
@@ -88,3 +88,8 @@ def add1(a):
         return a + 1
 
     assert sc.identical(add1(sc.scalar(1.0)), sc.scalar(2.0))
+
+
+def test_inplace_mod_for_float64():
```

Comment:
We have use the convention of mostly testing the C++ code with C++ tests, and only having minimal testing of the internals at the Python level.
So in this case, I think a better place for the test would be in this file: https://github.com/scipp/scipp/blob/main/lib/core/test/element_arithmetic_test.cpp

Reply:
Okay makes sense. I added a C++ test now. However, I was not able to compile and run the tests or the benchmarks.
The install build compiles.
Probably some version issue in my local environment.

```
$ cmake --build . --target all-tests
FAILED: bin/scipp-core-test 
...
/usr/bin/ld: /home/johkas/mambaforge/envs/scipp-dev/lib/libtbb.so.12.8: undefined reference to `std::__exception_ptr::exception_ptr::_M_release()@CXXABI_1.3.13'
```

Reply:
Strange I've not seen that error before.

Some threads suggest maybe something with a GCC version mismatch (10 and 11) between compiling and linking steps. If you installed `gcc` in the `scipp-dev` environment, could it possibly be clashing with a system installed GCC?

Reply:
From what I understand it uses the system installed GCC and the system installed linker. 🤔

<details>
<summary>Compile error message</summary>
<pre><code>
$ cmake --build . --target all-tests
...
[130/326] Linking CXX executable bin/scipp-core-test
FAILED: bin/scipp-core-test 
: && /usr/bin/c++ --param inline-unit-growth=50 -fno-omit-frame-pointer -g  lib/core/test/CMakeFiles/scipp-core-test.dir/array_to_string_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/dict_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/dimensions_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/eigen_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/element_array_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/element_array_view_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/element_arithmetic_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/element_bin_detail_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/element_comparison_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/element_event_operations_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/element_geometric_operations_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/element_histogram_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/element_logical_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/element_map_to_bins_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/element_math_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/element_special_values_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/element_to_unit_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/element_trigonometry_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/element_util_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/multi_index_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/slice_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/sizes_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/spatial_transforms_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/strides_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/string_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/subbin_sizes_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/time_point_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/value_and_variance_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/view_index_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/transform_common_test.cpp.o -o bin/scipp-core-test  -Wl,-rpath,/home/johkas/personal/scipp/scipp/build/lib/core:/home/johkas/personal/scipp/scipp/build/lib/units:/home/johkas/mambaforge/envs/scipp-dev/lib  lib/core/libscipp-core.so  lib/units/libscipp-units.so  /home/johkas/.conan/data/LLNL-Units/0.7.0/_/_/package/9f62ced26518f3e46410d0d887e36fd45aab971a/lib/libunits.a  /home/johkas/mambaforge/envs/scipp-dev/lib/libtbb.so.12.8  /home/johkas/.conan/data/gtest/1.11.0/_/_/package/8887312d8d5d906d59a60762be586000713d5f48/lib/libgtest_maind.a  /home/johkas/.conan/data/gtest/1.11.0/_/_/package/8887312d8d5d906d59a60762be586000713d5f48/lib/libgmock_maind.a  /home/johkas/.conan/data/gtest/1.11.0/_/_/package/8887312d8d5d906d59a60762be586000713d5f48/lib/libgmockd.a  /home/johkas/.conan/data/gtest/1.11.0/_/_/package/8887312d8d5d906d59a60762be586000713d5f48/lib/libgtestd.a  -lm  -lpthread && :
/usr/bin/ld: /home/johkas/mambaforge/envs/scipp-dev/lib/libtbb.so.12.8: undefined reference to `std::__exception_ptr::exception_ptr::_M_release()@CXXABI_1.3.13'
/usr/bin/ld: /home/johkas/mambaforge/envs/scipp-dev/lib/libtbb.so.12.8: undefined reference to `std::__exception_ptr::exception_ptr::_M_addref()@CXXABI_1.3.13'
collect2: error: ld returned 1 exit status
...
ninja: build stopped: subcommand failed.
</code></pre>
</details>

<details>
<summary>Complete ouput</summary>
<pre><code>
$ cmake --build . --target all-tests
[10/326] Building CXX object lib/core/CMakeFiles/scipp-core.dir/dimensions.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/rename.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/dimensions.cpp:8:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[11/326] Building CXX object lib/core/CMakeFiles/scipp-core.dir/dict.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dict.h:20,
                 from /home/johkas/personal/scipp/scipp/lib/core/dict.cpp:6:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[12/326] Building CXX object lib/core/CMakeFiles/scipp-core.dir/except.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/except.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[13/326] Building CXX object lib/core/CMakeFiles/scipp-core.dir/dtype.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/dtype.cpp:8:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[14/326] Building CXX object lib/core/CMakeFiles/scipp-core.dir/element_array_view.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element_array_view.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/element_array_view.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[15/326] Building CXX object lib/core/CMakeFiles/scipp-core.dir/multi_index.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/multi_index.h:12,
                 from /home/johkas/personal/scipp/scipp/lib/core/multi_index.cpp:6:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[16/326] Building CXX object lib/core/CMakeFiles/scipp-core.dir/sizes.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/rename.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/sizes.cpp:7:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[17/326] Building CXX object lib/core/CMakeFiles/scipp-core.dir/slice.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/slice.cpp:7:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[18/326] Building CXX object lib/core/CMakeFiles/scipp-core.dir/strides.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/strides.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/strides.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[19/326] Building CXX object lib/core/CMakeFiles/scipp-core.dir/string.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/string.cpp:12:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[20/326] Building CXX object lib/core/CMakeFiles/scipp-core.dir/view_index.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/view_index.h:9,
                 from /home/johkas/personal/scipp/scipp/lib/core/view_index.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[21/326] Building CXX object lib/core/CMakeFiles/scipp-core.dir/subbin_sizes.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/arithmetic.h:13,
                 from /home/johkas/personal/scipp/scipp/lib/core/subbin_sizes.cpp:9:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[22/326] Building CXX object lib/core/test/CMakeFiles/scipp-core-test.dir/dict_test.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dict.h:20,
                 from /home/johkas/personal/scipp/scipp/lib/core/test/dict_test.cpp:8:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[23/326] Building CXX object lib/core/test/CMakeFiles/scipp-core-test.dir/dimensions_test.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/test/dimensions_test.cpp:9:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[26/326] Building CXX object lib/core/test/CMakeFiles/scipp-core-test.dir/array_to_string_test.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/array_to_string.h:16,
                 from /home/johkas/personal/scipp/scipp/lib/core/test/array_to_string_test.cpp:11:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[27/326] Building CXX object lib/core/test/CMakeFiles/scipp-core-test.dir/element_arithmetic_test.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/arithmetic.h:13,
                 from /home/johkas/personal/scipp/scipp/lib/core/test/element_arithmetic_test.cpp:9:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[28/326] Building CXX object lib/core/test/CMakeFiles/scipp-core-test.dir/element_array_view_test.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element_array_view.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/test/element_array_view_test.cpp:10:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[29/326] Building CXX object lib/core/test/CMakeFiles/scipp-core-test.dir/element_bin_detail_test.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/bin_detail.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/test/element_bin_detail_test.cpp:4:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[30/326] Building CXX object lib/core/test/CMakeFil...p-core-test.dir/element_event_operations_test.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/histogram.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/event_operations.h:13,
                 from /home/johkas/personal/scipp/scipp/lib/core/test/element_event_operations_test.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[31/326] Building CXX object lib/core/test/CMakeFiles/scipp-core-test.dir/element_comparison_test.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/comparison.h:13,
                 from /home/johkas/personal/scipp/scipp/lib/core/test/element_comparison_test.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[32/326] Building CXX object lib/core/test/CMakeFil...re-test.dir/element_geometric_operations_test.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/geometric_operations.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/test/element_geometric_operations_test.cpp:7:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[33/326] Building CXX object lib/core/test/CMakeFiles/scipp-core-test.dir/element_histogram_test.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/util.h:14,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/histogram.h:13,
                 from /home/johkas/personal/scipp/scipp/lib/core/test/element_histogram_test.cpp:6:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[34/326] Building CXX object lib/core/test/CMakeFiles/scipp-core-test.dir/element_logical_test.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/logical.h:9,
                 from /home/johkas/personal/scipp/scipp/lib/core/test/element_logical_test.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[35/326] Building CXX object lib/core/test/CMakeFiles/scipp-core-test.dir/element_math_test.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/math.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/test/element_math_test.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[36/326] Building CXX object lib/core/test/CMakeFiles/scipp-core-test.dir/element_map_to_bins_test.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/util.h:14,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/map_to_bins.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/test/element_map_to_bins_test.cpp:4:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[37/326] Building CXX object lib/core/test/CMakeFil...ipp-core-test.dir/element_special_values_test.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/special_values.h:14,
                 from /home/johkas/personal/scipp/scipp/lib/core/test/element_special_values_test.cpp:6:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[38/326] Building CXX object lib/core/test/CMakeFiles/scipp-core-test.dir/multi_index_test.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element_array_view.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/test/multi_index_test.cpp:6:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[39/326] Building CXX object lib/core/test/CMakeFiles/scipp-core-test.dir/element_to_unit_test.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/to_unit.h:14,
                 from /home/johkas/personal/scipp/scipp/lib/core/test/element_to_unit_test.cpp:7:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[40/326] Building CXX object lib/core/test/CMakeFiles/scipp-core-test.dir/element_trigonometry_test.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/trigonometry.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/test/element_trigonometry_test.cpp:6:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[41/326] Building CXX object lib/core/test/CMakeFiles/scipp-core-test.dir/slice_test.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/test/slice_test.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[42/326] Building CXX object lib/core/test/CMakeFiles/scipp-core-test.dir/element_util_test.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/util.h:14,
                 from /home/johkas/personal/scipp/scipp/lib/core/test/element_util_test.cpp:8:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[43/326] Building CXX object lib/core/test/CMakeFiles/scipp-core-test.dir/sizes_test.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/test/sizes_test.cpp:7:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[44/326] Building CXX object lib/core/test/CMakeFiles/scipp-core-test.dir/strides_test.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/strides.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/test/strides_test.cpp:7:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[49/326] Building CXX object lib/core/test/CMakeFiles/scipp-core-test.dir/string_test.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/test/string_test.cpp:8:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[50/326] Building CXX object lib/core/test/CMakeFiles/scipp-core-test.dir/view_index_test.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/view_index.h:9,
                 from /home/johkas/personal/scipp/scipp/lib/core/test/view_index_test.cpp:7:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[51/326] Building CXX object lib/core/test/CMakeFiles/scipp-core-test.dir/transform_common_test.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/math.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/test/transform_common_test.cpp:4:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[52/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/abs.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/math.h:11,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/abs.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[53/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/exp.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/math.h:11,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/exp.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[54/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/log.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/math.h:11,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/log.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[55/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/log10.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/math.h:11,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/log10.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[56/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/reciprocal.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/math.h:11,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/reciprocal.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[57/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/norm.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/math.h:11,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/norm.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[58/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/sqrt.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/math.h:11,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/sqrt.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[59/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/ceil.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/math.h:11,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/ceil.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[60/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/rint.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/math.h:11,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/rint.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[61/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/floor.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/math.h:11,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/floor.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[62/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/erf.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/math.h:11,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/erf.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[63/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/erfc.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/math.h:11,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/erfc.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[64/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/dot.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/math.h:11,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/dot.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[65/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/cross.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/math.h:11,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/cross.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[66/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/stddevs.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/util.h:14,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/stddevs.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[67/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/variances.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/util.h:14,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/variances.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[68/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/sin.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/trigonometry.h:11,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/sin.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[69/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/values.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/util.h:14,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/values.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[70/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/cos.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/trigonometry.h:11,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/cos.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[71/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/asin.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/trigonometry.h:11,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/asin.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[73/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/tan.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/trigonometry.h:11,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/tan.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[74/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/acos.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/trigonometry.h:11,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/acos.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[75/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/atan.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/trigonometry.h:11,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/atan.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[76/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/atan2.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/trigonometry.h:11,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/atan2.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[77/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/isposinf.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/special_values.h:14,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/isposinf.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[79/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/isinf.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/special_values.h:14,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/isinf.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[80/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/isnan.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/special_values.h:14,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/isnan.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[81/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/isfinite.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/special_values.h:14,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/isfinite.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[82/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/isneginf.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/special_values.h:14,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/isneginf.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[83/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/equal.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/comparison.h:13,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/equal.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[84/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/greater_equal.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/comparison.h:13,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/greater_equal.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[85/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/greater.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/comparison.h:13,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/greater.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[86/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/less.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/comparison.h:13,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/less.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[87/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/less_equal.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/comparison.h:13,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/less_equal.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[88/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/not_equal.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/comparison.h:13,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/not_equal.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[89/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/floor_divide.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/arithmetic.h:13,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/floor_divide.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[90/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/mod.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/arithmetic.h:13,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/mod.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[91/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/negative.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/arithmetic.h:13,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/negative.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[92/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/logical_not.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/logical.h:9,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/logical_not.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[93/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/logical_or.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/logical.h:9,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/logical_or.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[94/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/mod_equals.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/arithmetic.h:13,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/mod_equals.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[95/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/logical_and.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/logical.h:9,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/logical_and.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[96/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/logical_xor.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/logical.h:9,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/logical_xor.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[97/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/logical_or_equals.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/logical.h:9,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/logical_or_equals.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[98/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/logical_and_equals.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/logical.h:9,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/logical_and_equals.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[99/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/logical_xor_equals.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/logical.h:9,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/logical_xor_equals.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[100/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/astype.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/tag_util.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/variable/astype.cpp:7:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[101/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/bin_util.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/variable.h:16,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/bin_util.h:7,
                 from /home/johkas/personal/scipp/scipp/lib/variable/bin_util.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[102/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/bins.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/variable.h:16,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/bins.h:7,
                 from /home/johkas/personal/scipp/scipp/lib/variable/bins.cpp:8:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[103/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/bin_array_variable.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/variable.h:16,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/include/scipp/variable/negative.h:8,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/include/scipp/variable/generated_arithmetic.h:7,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/arithmetic.h:8,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/bin_array_variable.tcc:5,
                 from /home/johkas/personal/scipp/scipp/lib/variable/bin_array_variable.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[104/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/bin_detail.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/bin_detail.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/variable/bin_detail.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[105/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/comparison.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/comparison.h:13,
                 from /home/johkas/personal/scipp/scipp/lib/variable/comparison.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[106/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/cumulative.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/variable.h:16,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/cumulative.h:8,
                 from /home/johkas/personal/scipp/scipp/lib/variable/cumulative.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[107/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/except.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/except.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/variable/except.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[108/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/creation.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/creation.h:14,
                 from /home/johkas/personal/scipp/scipp/lib/variable/creation.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[109/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/arithmetic.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/variable.h:16,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/include/scipp/variable/negative.h:8,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/include/scipp/variable/generated_arithmetic.h:7,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/arithmetic.h:8,
                 from /home/johkas/personal/scipp/scipp/lib/variable/arithmetic.cpp:3:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[110/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/pow.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/variable.h:16,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/pow.h:8,
                 from /home/johkas/personal/scipp/scipp/lib/variable/pow.cpp:6:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[111/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/operations.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/geometric_operations.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/variable/operations.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[112/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/math.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/variable.h:16,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/math.h:8,
                 from /home/johkas/personal/scipp/scipp/lib/variable/math.cpp:3:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[113/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/shape.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/variable/shape.cpp:7:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[114/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/rebin.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/util.h:14,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/rebin.h:13,
                 from /home/johkas/personal/scipp/scipp/lib/variable/rebin.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[115/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/reduction.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/variable.h:16,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/reduction.h:9,
                 from /home/johkas/personal/scipp/scipp/lib/variable/reduction.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[116/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/slice.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/variable.h:16,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/include/scipp/variable/equal.h:8,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/include/scipp/variable/generated_comparison.h:7,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/comparison.h:7,
                 from /home/johkas/personal/scipp/scipp/lib/variable/slice.cpp:8:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[117/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/sort.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/comparison.h:13,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/sort.h:9,
                 from /home/johkas/personal/scipp/scipp/lib/variable/sort.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[118/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/inv.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/variable.h:16,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/inv.h:7,
                 from /home/johkas/personal/scipp/scipp/lib/variable/inv.cpp:3:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[120/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/special_values.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/special_values.h:14,
                 from /home/johkas/personal/scipp/scipp/lib/variable/special_values.cpp:3:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[121/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/string.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/array_to_string.h:16,
                 from /home/johkas/personal/scipp/scipp/lib/variable/string.cpp:9:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[122/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/to_unit.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/string.h:17,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/except.h:15,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/transform_common.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/element/to_unit.h:14,
                 from /home/johkas/personal/scipp/scipp/lib/variable/to_unit.cpp:8:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[123/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/structures.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/variable.h:16,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/structures.h:7,
                 from /home/johkas/personal/scipp/scipp/lib/variable/structures.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[124/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/subspan_view.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/variable.h:16,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/subspan_view.h:7,
                 from /home/johkas/personal/scipp/scipp/lib/variable/subspan_view.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[125/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/util.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/variable.h:16,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/include/scipp/variable/values.h:8,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/include/scipp/variable/generated_util.h:7,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/util.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/variable/util.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[127/326] Building CXX object lib/variable/test/CMakeFiles/scipp-variable-test.dir/accumulate_test.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/variable.h:16,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/creation.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/shape.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/accumulate.h:7,
                 from /home/johkas/personal/scipp/scipp/lib/variable/test/accumulate_test.cpp:7:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[128/326] Building CXX object lib/variable/test/CMakeFiles/scipp-variable-test.dir/arithmetic_test.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/variable.h:16,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/include/scipp/variable/negative.h:8,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/include/scipp/variable/generated_arithmetic.h:7,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/arithmetic.h:8,
                 from /home/johkas/personal/scipp/scipp/lib/variable/test/arithmetic_test.cpp:8:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[129/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/variable_concept.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/variable_concept.h:9,
                 from /home/johkas/personal/scipp/scipp/lib/variable/variable_concept.cpp:1:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[130/326] Linking CXX executable bin/scipp-core-test
FAILED: bin/scipp-core-test 
: && /usr/bin/c++ --param inline-unit-growth=50 -fno-omit-frame-pointer -g  lib/core/test/CMakeFiles/scipp-core-test.dir/array_to_string_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/dict_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/dimensions_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/eigen_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/element_array_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/element_array_view_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/element_arithmetic_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/element_bin_detail_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/element_comparison_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/element_event_operations_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/element_geometric_operations_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/element_histogram_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/element_logical_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/element_map_to_bins_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/element_math_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/element_special_values_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/element_to_unit_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/element_trigonometry_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/element_util_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/multi_index_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/slice_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/sizes_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/spatial_transforms_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/strides_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/string_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/subbin_sizes_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/time_point_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/value_and_variance_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/view_index_test.cpp.o lib/core/test/CMakeFiles/scipp-core-test.dir/transform_common_test.cpp.o -o bin/scipp-core-test  -Wl,-rpath,/home/johkas/personal/scipp/scipp/build/lib/core:/home/johkas/personal/scipp/scipp/build/lib/units:/home/johkas/mambaforge/envs/scipp-dev/lib  lib/core/libscipp-core.so  lib/units/libscipp-units.so  /home/johkas/.conan/data/LLNL-Units/0.7.0/_/_/package/9f62ced26518f3e46410d0d887e36fd45aab971a/lib/libunits.a  /home/johkas/mambaforge/envs/scipp-dev/lib/libtbb.so.12.8  /home/johkas/.conan/data/gtest/1.11.0/_/_/package/8887312d8d5d906d59a60762be586000713d5f48/lib/libgtest_maind.a  /home/johkas/.conan/data/gtest/1.11.0/_/_/package/8887312d8d5d906d59a60762be586000713d5f48/lib/libgmock_maind.a  /home/johkas/.conan/data/gtest/1.11.0/_/_/package/8887312d8d5d906d59a60762be586000713d5f48/lib/libgmockd.a  /home/johkas/.conan/data/gtest/1.11.0/_/_/package/8887312d8d5d906d59a60762be586000713d5f48/lib/libgtestd.a  -lm  -lpthread && :
/usr/bin/ld: /home/johkas/mambaforge/envs/scipp-dev/lib/libtbb.so.12.8: undefined reference to `std::__exception_ptr::exception_ptr::_M_release()@CXXABI_1.3.13'
/usr/bin/ld: /home/johkas/mambaforge/envs/scipp-dev/lib/libtbb.so.12.8: undefined reference to `std::__exception_ptr::exception_ptr::_M_addref()@CXXABI_1.3.13'
collect2: error: ld returned 1 exit status
[131/326] Building CXX object lib/variable/test/CMakeFiles/scipp-variable-test.dir/astype_test.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/variable.h:16,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/astype.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/variable/test/astype_test.cpp:7:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[132/326] Building CXX object lib/variable/test/CMa.../scipp-variable-test.dir/bin_array_model_test.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/variable.h:16,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/include/scipp/variable/negative.h:8,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/include/scipp/variable/generated_arithmetic.h:7,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/arithmetic.h:8,
                 from /home/johkas/personal/scipp/scipp/lib/variable/test/bin_array_model_test.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[133/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/variable_factory.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/variable.h:16,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/variable_factory.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/variable/variable_factory.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[134/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/variable.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/variable.h:16,
                 from /home/johkas/personal/scipp/scipp/lib/variable/variable.cpp:7:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[135/326] Building CXX object lib/variable/CMakeFil...ariable.dir/variable_instantiate_bin_elements.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/variable.h:16,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/include/scipp/variable/negative.h:8,
                 from /home/johkas/personal/scipp/scipp/build/lib/variable/include/scipp/variable/generated_arithmetic.h:7,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/arithmetic.h:8,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/bin_array_variable.tcc:5,
                 from /home/johkas/personal/scipp/scipp/lib/variable/variable_instantiate_bin_elements.cpp:5:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[136/326] Building CXX object lib/variable/CMakeFiles/scipp-variable.dir/variable_instantiate_basic.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/element_array_model.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/element_array_variable.tcc:5,
                 from /home/johkas/personal/scipp/scipp/lib/variable/variable_instantiate_basic.cpp:7:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[137/326] Building CXX object lib/variable/CMakeFil...cipp-variable.dir/variable_instantiate_linalg.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/variable.h:16,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/bins.h:7,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/structure_array_variable.tcc:5,
                 from /home/johkas/personal/scipp/scipp/lib/variable/variable_instantiate_linalg.cpp:7:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[138/326] Building CXX object lib/variable/CMakeFil...riable.dir/variable_instantiate_view_elements.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/element_array_model.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/element_array_variable.tcc:5,
                 from /home/johkas/personal/scipp/scipp/lib/variable/variable_instantiate_view_elements.cpp:6:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
[139/326] Building CXX object lib/variable/CMakeFil...ariable.dir/variable_instantiate_map_elements.cpp.o
In file included from /home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/dimensions.h:11,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/element_array_model.h:10,
                 from /home/johkas/personal/scipp/scipp/lib/variable/include/scipp/variable/element_array_variable.tcc:5,
                 from /home/johkas/personal/scipp/scipp/lib/variable/variable_instantiate_map_elements.cpp:9:
/home/johkas/personal/scipp/scipp/lib/core/include/scipp/core/sizes.h:11:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
   11 | #pragma GCC diagnostic ignored "-Wstringop-overread"
      |                                ^~~~~~~~~~~~~~~~~~~~~
cc1plus: warning: unrecognized command line option ‘-Wno-unknown-warning-option’
ninja: build stopped: subcommand failed.
</code></pre>
</details>

```
$ /usr/bin/c++ --version
c++ (Ubuntu 9.4.0-1ubuntu1~20.04.1) 9.4.0
```

```
$ /usr/bin/ld --version
GNU ld (GNU Binutils for Ubuntu) 2.34
```

Reply:
FWIW the compile issue can be reproduced in a Docker image using the following `Dockerfile`:

```dockerfile
FROM mambaorg/micromamba

# Install git and gcc
USER root
RUN apt-get update \
 && apt-get upgrade -y \
 && apt-get install -y git build-essential

USER mambauser

# Download scipp
WORKDIR /scipp
RUN git clone https://github.com/scipp/scipp.git \
 && cd scipp \
 && git submodule init \
 && git submodule update
WORKDIR /scipp/scipp

# Create the environment
ENV ENV_NAME=scipp-dev
RUN micromamba create -y -f docs/environments/developer.yml \
 && micromamba clean --all --yes

# Installing a compiler in the env resolves the issue
#RUN micromamba install -y -n $ENV_NAME -c conda-forge cxx-compiler openmpi \
# && micromamba clean --all --yes

ARG MAMBA_DOCKERFILE_ACTIVATE=1

RUN mkdir build install

WORKDIR /scipp/scipp/build
RUN cmake \
  -GNinja \
  -DCMAKE_BUILD_TYPE=Debug \
  -DPython_EXECUTABLE=$(command -v python3) \
  -DCMAKE_INSTALL_PREFIX=../install \
  -DCMAKE_INTERPROCEDURAL_OPTIMIZATION=OFF \
  -DDYNAMIC_LIB=ON \
  ..
RUN cmake --build . --target all-tests
RUN cmake --build . --target install

# Run tests tests
WORKDIR /scipp/scipp
CMD ctest --preset test \
 && PYTHONPATH="./install:$PYTHONPATH" python -m pytest tests
```

Like you said, the issue seems to be a collision with the system toolchain. Adding a compiler + openmpi to the mamba env resolved the issue.

Reply:
Thanks for the info and for going the extra mile using Docker